estate/models/estate.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from dateutil.relativedelta import *
+
+
+class Estate(models.Model):
+    _name = "estate_property"
+    _description = "RE Initial Model"
+    name = fields.Char(required=True)
```

Comment:
Use a space to separate this
```suggestion
    _description = "RE Initial Model"

    name = fields.Char(required=True)
```